### DISCUSSION TITLE
License Non-Compliance of some iOS apps

### DISCUSSION THREAD

**Discussion by nguyenpham** [official-stockfish/Stockfish#4855] [General]
When finding some information to write a wiki page, I realize that some iOS apps don't respect the Stockfish license, which requires all software embedding its code to be open source and publish links to their source code.

Just list a few iOS apps after a quick search. I am sure the full list must be much longer. They all say clearly they are using Stockfish:

- SmallFish Chess for Stockfish: it is a huge surprise since this app is considered an official one when its name is published on the Stockfish homepage
- Chess Move - Stockfish Engine
- Syzygy Tablebase for Stockfish
- Chessvision.ai Chess Scanner
- Analyze This Chess
- Arkon: Chess Opening Explorer
- Imperial Chess
- iPlay Chess
- Pocket Chess Book
- Ulimate Chess League

Correct me if you find any wrong. Add more if you could.

**Comment by Craftyawesome** [comment]
I can't speak for literally anything else, but from looking at the screenshots it looks like chessvision just opens lichess to analyze, which should have everything correctly handled.
